From 1a48d30f0ae9fb808648c3a1d4dde074ef1fe2b3 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Wed, 8 Jul 2020 01:15:20 +0200
Subject: [PATCH 114/346] Always enable GPU regulators

see CAJiuCcf8fi_w0-Xo=MMMv=5jBqcS68jEbjbo4NvVR24b4BuZHg@mail.gmail.com

> This patch is normally no more required since this serie:
> https://lore.kernel.org/linux-pm/cover.1589528491.git.viresh.kumar@linaro.org/

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi-3.dts | 1 +
 arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi.dtsi  | 1 +
 arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts   | 1 +
 arch/arm64/boot/dts/allwinner/sun50i-h6-tanix-tx6.dts  | 1 +
 4 files changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi-3.dts b/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi-3.dts
index 75f10da09531..ffd3d811511f 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi-3.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi-3.dts
@@ -308,6 +308,7 @@ reg_dcdcc: dcdcc {
 				regulator-max-microvolt = <1080000>;
 				regulator-ramp-delay = <2500>;
 				regulator-name = "vdd-gpu";
+				regulator-always-on;
 			};
 
 			reg_dcdcd: dcdcd {
diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi.dtsi
index ebc120a9232f..1cc0d2ab8e2b 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi.dtsi
@@ -209,6 +209,7 @@ reg_dcdcc: dcdcc {
 				regulator-min-microvolt = <810000>;
 				regulator-max-microvolt = <1080000>;
 				regulator-name = "vdd-gpu";
+				regulator-always-on;
 			};
 
 			reg_dcdcd: dcdcd {
diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts b/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts
index af85b2074867..a79c6fafabf7 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts
@@ -254,6 +254,7 @@ reg_dcdcc: dcdcc {
 				regulator-max-microvolt = <1080000>;
 				regulator-ramp-delay = <2500>;
 				regulator-name = "vdd-gpu";
+				regulator-always-on;
 			};
 
 			reg_dcdcd: dcdcd {
diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h6-tanix-tx6.dts b/arch/arm64/boot/dts/allwinner/sun50i-h6-tanix-tx6.dts
index be81330db14f..18cb59ccd96d 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h6-tanix-tx6.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h6-tanix-tx6.dts
@@ -44,6 +44,7 @@ reg_vdd_cpu_gpu: vdd-cpu-gpu {
 		regulator-name = "vdd-cpu-gpu";
 		regulator-min-microvolt = <1135000>;
 		regulator-max-microvolt = <1135000>;
+		regulator-always-on;
 	};
 };
 
-- 
2.25.1

